tests/test_instance.py

```diff
@@ -13,6 +14,34 @@
 from conftest import CREATE_INSTANCE_METHOD, DELETE_INSTANCE_METHOD, GET_INSTANCE_METHOD
 
 
+@pytest.fixture(scope="session", autouse=True, params=[True, False])
+def configuration_mock(request):
+    pytest.isSecure = request.param
```

Comment:
For test that have branching based on whether the channel is secure or not, you should rather split them (`test_create_secure_channel`, `test_create_insecure_channel`)